tasks/managed/update-trusted-tasks/README.md

```diff
@@ -10,3 +10,7 @@ If it is already in place, it will be used as an input to which the results will
 | Name       | Description                                                             | Optional | Default value |
 |------------|-------------------------------------------------------------------------|----------|---------------|
 | snapshotPath   | Path to the JSON string of the Snapshot spec in the data workspace | No       | -             |
+
+## Changes in 0.2.0
+
```

Comment:
My suggestion was to remove the empty line too ;) BTW, according to https://status.redhat.com Quay is still having issues, so e2e can't run now anyway.